setup.pl

```diff
@@ -609,11 +618,25 @@
 	print RELOAD "kill -USR1 \`cat \$pidfile\`\n";
 	close(RELOAD);
 
+	# Pre install
+	open(PREINST, ">$config_directory/.pre-install");
```

Comment:
N: It just generates default scripts for pre and post install commands, and in this section does it for older systems to use your code from `.stop-init` and `.start-init` scripts.